web/chat/chat-tabs.react.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import ChatThreadTab from './chat-thread-tab.react';
+import css from './chat-tabs.css';
+
+type Props = {|
+  children: Array<React.Element<any>>,
```

Comment:
Maybe make it more specific: `children: Array<React.Element<typeof ThreadsTab>>`?